namereg/namereg.sol

```diff
@@ -4,25 +4,23 @@
 //   Gav Wood <g@ethdev.com>
 
 contract NameRegister {
-	function getAddress(string32 _name) constant returns (address o_owner) {}
-	function getName(address _owner) constant returns (string32 o_name) {}
+	function getAddress(bytes32 _name) constant returns (address o_owner) {}
+	function getName(address _owner) constant returns (bytes32 o_name) {}
 }
 
-#require service, owned
+import "service";
+import "owned";
 contract NameReg is service(1), owned, NameRegister {
   	event AddressRegistered(address indexed account);
   	event AddressDeregistered(address indexed account);
 
 	function NameReg() {
-		toName[Config()] = "Config";
```

Comment:
I don't know, I just wanted to be able to compile this contract. It was failing with following error:
```
namereg.sol:18:16: Type error: More than one argument for explicit type conversion.
        toName[Config()] = "Config";
               ^------^
```